packages/templates/src/raw/fresh-and-bright/template.json

```diff
@@ -1623,7 +1683,29 @@
       "type": "page",
       "id": "9845d2e8-8afe-448b-b7c7-ed1b5e30ab69",
       "backgroundColor": { "color": { "r": 255, "g": 255, "b": 255 } },
-      "pageTemplateType": "editorial"
+      "pageTemplateType": "editorial",
+      "groups": {
+        "0f035801-d8bb-4a82-ae00-0cba807b6ae6": {
+          "name": "Action",
+          "isLocked": false,
+          "isCollapsed": true
+        },
+        "235f4576-d429-41e8-9823-543901200926": {
+          "name": "Action",
```

Comment:
I believe Pea Pod was mainly working on templates and not sure if there's anyone to ask but perhaps @swissspidy might know:
Looks like in this template there are various cases of buttons created by a text and a rectangle behind it. Any idea if there's a reason why a text with background color isn't used here instead to have better centering and just one element? 🤔
 It's grouped in this PR, however, having a text with background might make more sense?
![Screenshot 2022-07-25 at 18 45 40](https://user-images.githubusercontent.com/3294597/180820492-6f682830-0b65-448c-b866-a13ff64b19eb.png)

Reply:
The templates were originally created by another agency.

It's possible that they were not aware of the background color feature or that it didn't fit the needs for some reason.

Agreed that having a text with background makes more sense in cases like this 👍